style.css

```diff
@@ -0,0 +1,131 @@
+* {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  color: #344563;
+}
+
+body {
+  width: fit-content;
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-x: hidden;
+}
+
+header {
+  background-color: white;
+  width: 100%;
+}
+
+.backgroundimg {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/image/background.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: white;
+  border-radius: 0 0 0 120px;
+  height: 680px;
+  width: 100%;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+}
+
+.img {
+  width: 100%;
+}
+
+.navmenu {
+  display: flex;
+  justify-content: space-between;
+  margin: 8px 18px;
+  align-items: center;
+  width: 375px;
+  height: 48px;
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+nav img {
+  margin-right: 50px;
+  display: grid;
+  width: 25px;
+}
+
+.introduction {
+  margin-top: 100px;
+  width: 300px;
+  height: 300px 0 0 0;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.backgroundimg .menuSection {
+  width: 60%;
+}
```

Comment:
- [ ] Please consider removing this as it's currently making your design not look perfect. This essentially makes the width of your design consistent. 


```
To be removed.

.backgroundimg .menuSection {
  width: 60%;
}
```